media/inf/scsi.inf

```diff
@@ -26,6 +26,7 @@ DefaultDestDir = 12
 %GenericMfg% = GenericMfg
 
 [GenericMfg]
+%GEN_SCSIADAPTER.DeviceDesc% = NO_DRV,,GEN_SCSIADAPTER
 %PCI\VEN_104B&CC_0100.DeviceDesc% = BusLogic_Inst,PCI\VEN_104B&CC_0100
```

Comment:
I'd say this is not a place for BusLogic driver

Reply:
Agree, it should be in driver directory INF file, but it's out of this PR scope, so for the later.